tests/integration/cqlengine/query/test_limit.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2013-2017 DataStax, Inc.
```

Comment:
There's no need to create a new file, this can be included in the file `test_queryset`